network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -30,13 +30,27 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
 
 {% block guts %}
 <div class="project-list-section">
-  <div class="creepiness-slider bg-white text-center">
+  <div id="sticky-bar" class="creepiness-slider bg-white text-center">
+
+    <!-- temporary location for dev work -->
+    <span id="product-filter-search" tabindex="0">
+      <span class="search-icon">&nbsp;</span>
+      <input type="text" role="searchbox" required id="product-filter-search-input" placeholder="Search product name..." class="body-large">
```

Comment:
I'm thinking we should remove `required` from here since this is not for a form submission and an empty value here should be considered as valid input. This way we also don't need extra CSS to hide the error outline.

```suggestion
      <input type="text" role="searchbox" id="product-filter-search-input" placeholder="Search product name..." class="body-large">
```

Reply:
the `required` is necessary in order to make use of the CSS :valid and :invalid pseudo-selectors. We can take it out in follow-up and do more on the JS side, but that requires quite some logic rewriting, and is not something I want to do this close to a deadline when we're not done with the rest yet.

Reply:
yea sounds fine to me